documentation/modules/con-brokered-topic.adoc

```diff
@@ -9,6 +9,8 @@
 The topic address type supports the publish-subscribe messaging pattern in which there are 1..N producers and 1..M consumers. Each message published to a topic address is forwarded to all subscribers for that address. A subscriber can also be durable, in which case messages are kept until the subscriber has acknowledged them.
 // !brokered.address.topic.longDescription:stop
 
+NOTE: Because subscriptions created after messages have been sent to the topic do not currently work properly, you must create the subscription before you create the topic.
```

Comment:
We don't define a subscription type in the brokered space, so this can be removed.

Reply:
Removed